packages/myst-to-jats/src/index.ts

```diff
@@ -393,6 +408,10 @@ const plugin: Plugin<[Options?], Root, VFile> = function (opts) {
     const tree = copyNode(node) as any;
     basicTransformations(tree, file);
     const state = new JatsSerializer(file, opts ?? { handlers });
+    const footnotes = selectAll('footnoteDefinition', tree) as FootnoteDefinition[];
```

Comment:
We handle `footnoteDefinition` nodes left in the tree here. This is nice and what we should move to everywhere...

Reply:
Nice, looking forward to that.